### ISSUE TITLE
Rclone not sleeping when hits limit

### ISSUE THREAD

**Issue by sbcrumb** [l3uddz/cloudplow#25]
It doesn't sleep it just sits there and keeps trying to upload even though it is not going logs just show the files it is trying to upload and all sitting at 0% no mention of the limit or the sleep or the error
"Failed to copy: googleapi: Error 403: User rate limit exceeded

**Comment by Superduper09** [comment]
Adding a trigger for "0/s" worked well for me to fix this. After using it for a couple weeks it works great. I rarely see the google api errors but the 0/s is flawless. Lowered my sleep timer check because my gdrive resets at midnight daily.



            "rclone_sleeps": {
                "Failed to copy: googleapi: Error 403: User rate limit exceeded": {
                    "count": 2,
                    "sleep": 2,
                    "timeout": 600
                },
                "0/s": {
                    "count": 10,
                    "sleep": 2,
                    "timeout": 600
                }
            },

**Comment by sbcrumb** [comment]
ok let me give that a try see if that will work.

**Comment by Superduper09** [comment]
Yeah the verbose 2 is definitely a better fix, do you think it impacts performance? That is why I initially played with the "0/s".

Just today though with lots of smaller files (<20mb) I was getting 0/s on some of them either before starting upload or after 100% finished, even though uploading was fine and wasn't blocked etc.. so I removed that trigger for now... I'll test some more when I'm blocked soon :)

**Comment by desimaniac** [comment]
Hmm. Apparently the "0/s" is being triggered by just 0s. Perhaps the slash needs to be escaped. I'd just stick with the phrase since it's more specific and you could even increase the number it counts, if you wanted, to give it more time before quitting.
